### DISCUSSION TITLE
Kerberos and AD

### DISCUSSION THREAD

**Discussion by GrumpyMailImporter** [ltb-project/self-service-password#783] [Q&A] [answered]
Will Kerberos authentication work for changing Active Directory passwords or does LDAPS with certificates need to be used?

**Comment by coudot** [comment, marked as answer]
The product currently only support LDAP protocol to change passwords

**Reply by GrumpyMailImporter** [reply]
Kerberos is mentioned in the LDAP connection setup documentation though?

**Reply by coudot** [reply]
Indeed, you use Kerberos to authenticate, to avoid storing the service account password in the configuration file. Kerberos cannot be used to update the password.

**Reply by GrumpyMailImporter** [reply]
Right, so if Kerberos is used as the authentication method, then users should be able to use the portal to change their AD password?

**Reply by GrumpyMailImporter** [reply]
What I'd like to do is change Active Directory passwords without having to install certificates. Would using Kerberos achieve that goal or would certificates with secure LDAP still be required?

**Reply by coudot** [reply]
Installing CA certificate is easy.

Again, you can't use Kerberos to change passwords, so forget this option, unless you code it by yourself.
